src/v/debug_bundle/debug_bundle_service.cc

```diff
@@ -225,8 +225,11 @@ class service::debug_bundle_process {
         vassert(
           _rpk_process != nullptr,
           "RPK process should be created if calling wait()");
+        auto set_finish_time = ss::defer(
+          [this] { _finished_time = clock::now(); });
         try {
-            co_return _wait_result.emplace(co_await _rpk_process->wait());
+            auto res = _wait_result.emplace(co_await _rpk_process->wait());
+            co_return res;
```

Comment:
Was this change intended?

Reply:
Oh i wonder if during development I had something "in-between" these lines... so yeah that can change lol